.circleci/config.yml

```diff
@@ -34,6 +34,7 @@ jobs:
 
     steps:
     - setup
+    - checkout
```

Comment:
checkout is already contain in the `setup` command

Reply:
right, thanks.